webapp/sources/rudder/rudder-web/src/main/scala/com/normation/rudder/web/services/LogDisplayer.scala

```diff
@@ -92,18 +93,36 @@ class LogDisplayer(
       (for {
         parsed   <- tryo(parse(jsonInterval)) ?~! s"Error when trying to parse '${jsonInterval}' as a JSON datastructure with fields 'start' and 'end'"
         startStr <- parsed \ "start" match {
-                      case JString(startStr) => tryo(DateTime.parse(startStr, format)) ?~! s"Bad format for start date, was execpting '${format.toString}' and got '${startStr}'"
-                      case x => Failure("Error: missing start date and time")
-                    }
+          case JString(startStr) if startStr.nonEmpty =>
```

Comment:
can you indent it like it was before , two space after parsed

Reply:
First case must be:
case JString("") => Full(None)
case JString(startStr) with no if => 
case _ => Failure("invalid value for start date")